src/stores/right-panel/RightPanelStore.ts

```diff
@@ -433,6 +435,20 @@ export default class RightPanelStore extends ReadyWatchingStore {
                 }
                 break;
             }
+            case Action.ViewRoom: {
```

Comment:
shouldn't we be responding to RVS changes here instead?

Reply:
(otherwise presumably clicking a permalink/etc will also clear out the right panel history)

Reply:
yeah, fixing